02-esc/modal.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Book Challenges</title>
+    <link rel="stylesheet" type="text/css" href="static/main.css">
+</head>
+<body class="modal">
+    
```

Comment:
Hela det här dokumentet är en placeholder och behöver inte komma med i PR.

Reply:
Precis. Denna skrev jag först innan jag la in det i javascriptfilen. Har raderat den.